src/domain/profile/__tests__/ProfileProjectDropdown.test.jsx

```diff
@@ -38,50 +38,56 @@ const getProjects = (projects) => ({
 
 describe('<ProfileProjectDropdown />', () => {
   beforeEach(() => {
-    jest.spyOn(projectService, 'projectId', 'get').mockReturnValue('1');
+    vi.spyOn(projectService, 'projectId', 'get').mockReturnValue('1');
   });
 
   it('should render null when loading', () => {
-    jest.spyOn(ReactQuery, 'useQuery').mockReturnValueOnce({ isLoading: true });
+    vi.spyOn(ReactQuery, 'useQuery').mockReturnValueOnce({ isLoading: true });
 
     const { container } = getWrapper();
 
     expect(container.childNodes.length).toEqual(0);
   });
 
-  it('should render null when error', () => {
-    jest
-      .spyOn(ReactQuery, 'useQuery')
-      .mockReturnValueOnce({ isLoading: false, error: new Error('Test') });
+  it('should render null when error', async () => {
+    vi.spyOn(ReactQuery, 'useQuery').mockReturnValueOnce({
+      isLoading: false,
+      error: new Error('Test'),
+    });
 
     const { container } = getWrapper();
-
-    expect(container.childNodes.length).toEqual(0);
+    await React.act(async () => {
+      await waitFor(() => {
+        expect(container.childNodes.length).toEqual(0);
+      });
```

Comment:
Probably the wrapping by React.act here is not necessary and doesn't cause any warnings